Exercise_C/index.css

```diff
@@ -1,4 +1,30 @@
-/* 
-This css sheet is linked to index.html
-You can style tags, classes (.) or ids (#) from this file.
-*/
+#container {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    align-items: flex-start;
+    margin: 10px;
+    height: 200px;
+    width: 600px;
+}
+
+#blue {
+    height: 100px;
+    width: 100px;
+    background-color: blue;
+}
+
```

Comment:
Great use of order to re-create the image!

Same comment about saving some lines through by removing the height and width from each id.